example_vulnerable_code.py

```
@@ -0,0 +1,70 @@
+
+# Security Issue 4: Path traversal vulnerability
+def read_file(user_input):
+    # No validation - path traversal risk
+    with open(f"/var/data/{user_input}", 'r') as f:
+        return f.read()
+
+# Code Quality Issue 1: Broad exception handling
+def risky_operation():
+    try:
+        # Some complex operation
+        result = 1 / 0
```

Ruff (0.11.9)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)